witness/src/Dscp/Resource/Keys/Error.hs

```diff
@@ -25,8 +25,10 @@ instance Show KeyInitError where
 
 instance Buildable KeyInitError where
     build = \case
-        SecretWrongPassPhraseError password ->
-            "Wrong password for educator key storage provided ("+|password|+")"
+        SecretDecryptionError msg ->
+            "Wrong password or malformed educator key provided ("+|msg|+
+            "). Either retype password or" +|
+            "make sure educator.key file corresponds to the right account"
```

Comment:
`Buildable` instance for `DecryptionError` already provides adequate explanation of the error happened. It should be just `"Error while decrypting educator key: "+|msg`

Reply:
Should we leave actionable feedback part of the error message?

Reply:
Let it be.